blog/docker_tooling_eclipse_mars.adoc

```diff
@@ -0,0 +1,36 @@
+= Screencast of Docker Tooling for Eclipse
+:page-date: 2015-07-01
+:page-layout: blog
+:page-author: xcoulon
+:page-tags: [docker, wildfly, jbosscentral]
+
+Last week at http://devnation.org[DevNation] I had the chance to show some of the cool features that we
+provide in JBoss Developer Studio, including the Docker tooling that we've built for Eclipse Mars
+and that was released last weekfootnote:[Docker tooling is part of Developer Studio since version 9.0.0.Beta1 and JBoss Tools users can install it from https://marketplace.eclipse.org/content/jboss-tools-mars[Eclipse Marketplace] or from http://download.jboss.org/jbosstools/mars/development/updates/[our Update site].].
+
+I also recorded a screencast that highlights some of those Docker tooling features.
+The video shows how to:
+
+ * Connect to a local Docker engine (0:32)
+ * Start a PostgreSQL container with a custom username and password (1:52)
+ * Build a custom image for Wildfly to include the PostgreSQL JDBC driver and configure a datasource (3:20)
+ * Start a WildFly container linked to PostgreSQL, with exposed ports and a volume mounted on the host machine (6:44)
+ * Import a Quickstart project using JBoss Central (8:30)
+ * Deploy the project on the WildFly container using a Server Adapter with custom settings (9:55)
+ * Connect to the JVM using the Remote Debugging tools (13:25)
+
+video::rzh-95FA6NA[youtube]
+
+
+I commented all the tasks as I recorded this screencast to highlight the important steps, but you'll have to cope with my accent ;-)
+Beware that if you run 'boot2docker', you'll need to use a specific IP address instead of 'localhost' when creating the Server adapter in Eclipse (trust me, I made this mistake before...)
```

Comment:
huh ? not following you here: I'm specificaly mentioning the case of boot2docker. Granted, having 'dockerhost' is a workaround.